### ISSUE TITLE
Should we adopt camel case for object names?

### ISSUE THREAD

**Issue by alexmlucas** [matthewscharles/instrument-maker#55]
Might improve readability, i.e. [im.envelopeFollower], [im.keyRelease]

**Comment by matthewscharles** [comment]
makes sense - this is the format for parameters/max-style attributes I have lined up e.g. delayTime:1000 

I'd like to find out what people think about the accessibility of this. great way to introduce the concept, but does it create a barrier e.g. with initial pick-up of the idea or using an on-screen keyboard?  

I've also broken object names up with dots in the past in which case the word division also indicates a hierarchy category name e.g.

im.key.press
im.key.hold
im.key.release
im.envelope.follower
im.envelope.generator

Robyn likened this to ip addresses
What do you think?

I would probably prefer to keep duplicates of the objects lower and camelcase which could be created automatically by a compiler script, but it could get messy

**Comment by matthewscharles** [comment]
I've been considering for some time dropping the im prefix and naming the folder im
so the user could type 

im/keypress or keypress 
similarly this could also become im/key/press

(or indeed im/keyPress, im/key.press)
